source/retryable-reads/tests/unified/handshakeError.yml

```diff
@@ -0,0 +1,68 @@
+description: "retryable reads handshake failures"
+
+schemaVersion: "1.0"
```

Comment:
`runOnRequirement.auth`, the "load-balanced" topology, and CMAP observation were introduced in schema version 1.3 (https://github.com/mongodb/specifications/commit/ed4e62b48939378b814cbd7d9d4939a2c1076a6b).

Reply:
done!